app/code/Magento/Checkout/view/frontend/web/template/minicart/item/default.html

```diff
@@ -88,8 +88,7 @@
                            'data-cart-item': item_id,
                            title: $t('Update')
                            }"
-                            class="update-cart-item"
-                            style="display: none">
+                            class="update-cart-item">
                         <span data-bind="i18n: 'Update'"></span>
```

Comment:
Now button update in minicart always show by default ?

Reply:
yeah... feels like it just having opacity 0...

Reply:
Can we update for luma theme also ? opacity only apply for blank theme

Reply:
> Hi @martasiewierska Thanks for you contribution. But I'm not sure about
> 
> > big chunk of code
> 
> Described in this PR ui modules included as separate files
> https://github.com/magento/magento2/blob/2.4-develop/lib/web/jquery/ui-modules/effect-blind.js (2.35KB) and https://github.com/magento/magento2/blob/2.4-develop/lib/web/jquery/ui-modules/effect-fade.js (660Bytes)
> Unfortunately end user will not save huge traffic and see performance effect.

Please note that these files also seems to load the file https://github.com/magento/magento2/blob/2.4-develop/lib/web/jquery/ui-modules/effect.js (46.7 KB).

With the changes in this PR, this file will not be loaded anymore, and so we will save around 50 KB total.